src/main/kotlin/day05/GoodPerson.kt

```
@@ -0,0 +1,25 @@
+package day05
+/*연습문제 2 에서 작성한 Person 클래스로 생성한 ‘홍길동’, ‘한석봉'의 나이를 각각 20, 25살 이라고 할 때,
+이름과 나이를 쌍으로 적당한 컬렉션에 넣습니다.-> map
+그 다음, 컬렉션에 저장한 값을 하나씩 다음과 같이 출력합니다.
+“홍길동의 나이는 20살”
+“한석봉의 나이는 25살”
+*/
+class GreatMan(var name: String, var age: Int) {
+
+}
```

detekt (1.23.7)

[warning] The class or object GreatMan is empty.

(detekt.empty-blocks.EmptyClassBlock)